src/tree/azureAccountTreeItem.ts

```diff
@@ -200,5 +201,17 @@ function getSubscriptionContext(
         userId: session.account.id,
         environment,
         isCustomCloud: environment.name === "AzureCustomCloud",
+        createCredentialsForScopes: async (scopes: string[]): Promise<TokenCredential> => {
+            const authSession = await sessionProvider.getAuthSession({ scopes });
+            if (failed(authSession)) {
+                throw new Error(`No Microsoft authentication session found: ${authSession.error}`);
+            }
+
+            return {
+                getToken: async () => {
+                    return { token: authSession.result.accessToken, expiresOnTimestamp: 0 };
```

Comment:
so expiresOnTimestamp: 0  means this token never expires?

Reply:
I believe that's the case, not documented outright but see its usage in this repo here: https://github.com/Azure/vscode-aks-tools/blob/main/src/auth/azureSessionProvider.ts#L288

Reply:
I tried setting the expiration time as 5 minutes but it didn't affect anything. But the official docs [here](https://learn.microsoft.com/en-us/javascript/api/@azure/core-auth/accesstoken?view=azure-node-latest#@azure-core-auth-accesstoken-expiresontimestamp) says it's the token expiration time. I think we need explore more on this. But for this PR it should be fine.